Components/input/input.css

```diff
@@ -0,0 +1,44 @@
+.inp-container {
+  margin: 1rem;
+  position: relative;
+}
+
+.inp-label {
+  letter-spacing: 0.025rem;
+  color: #393d3d;
```

Comment:
Create variables for the colors.

Reply:
Yes vrinda I will once I start with documentation site , for now I have not decided my colors

Reply:
Thanks for the review